### PR TITLE
Updated TextInput.jsx for text color field.

### LINKED ISSUE #4183: Add text color style in text input widget
## Proposed feature
Option to change and update the text color of the text input widget with the `Text color` property in the style tab.

### Description
Currently, the text input widget only supports **Border radius**, **Visibility**, and **Disable** as style definitions.

Screenshot 2022-10-01 at 10 19 24 AM

reference: https://github.com/ToolJet/ToolJet/pull/1721

### PR DESCRIPTION
**Pull request by dmdbilal**
fixes #4183

With the reference of the issue #4183. I've resolved this issue by adding the missing codes in the TextInput.jsx file.
@arpitnath

### CODE REVIEW

**Review by arpitnath — CHANGES_REQUESTED** [review]
It will have compile errors.
Can you please, check the UI after making changes?

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 1]
```diff
@@ -879,18 +879,37 @@ export const widgets = [
...
       },
-      visibility: { type: 'toggle', displayName: 'Visibility', validation: { schema: { type: 'boolean' } } },
       disabledState: { type: 'toggle', displayName: 'Disable', validation: { schema: { type: 'boolean' } } },
     },
     exposedVariables: {
       value: '',
+      visibility: { value: '{{true}}' },
+      disabledState: { value: '{{false}}' },
+      borderRadius: { value: '{{0}}' },
+      textColor: { value: '#232e3c' },
```
Comment (arpitnath) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:912]: why are we exposing the values?

**Review by arpitnath — CHANGES_REQUESTED** [review]

File: frontend/src/Editor/WidgetManager/widgetConfig.js (hunk 1 above)
Comment (dmdbilal) [reply frontend/src/Editor/WidgetManager/widgetConfig.js]: OOPS ! Sorry misplaced.

**Comment by dmdbilal** [comment]
Made changes @arpitnath.

File: frontend/src/Editor/Components/TextInput.jsx [hunk 2]
```diff
@@ -39,6 +42,15 @@ export const TextInput = function TextInput({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [properties.value]);
 
+  const computedStyles = {
```
Comment (arpitnath) [inline frontend/src/Editor/Components/TextInput.jsx:45]: variables : `computedStyles`, `disabledState` are never being used.

File: frontend/src/Editor/Components/TextInput.jsx [hunk 3]
```diff
@@ -39,6 +42,15 @@ export const TextInput = function TextInput({
...
+    backgroundColor,
```
Comment (arpitnath) [inline frontend/src/Editor/Components/TextInput.jsx:46]: crashing the widget with a `ReferenceError`.
`backgroundColor` is not defined.

File: frontend/src/Editor/Components/TextInput.jsx [hunk 4]
```diff
@@ -94,9 +106,8 @@ export const TextInput = function TextInput({
           fireEvent('onFocus');
         }}
         type="text"
-        className={`form-control ${!isValid ? 'is-invalid' : ''} validation-without-icon ${
-          darkMode && 'dark-theme-placeholder'
-        }`}
+        className={`form-control ${!isValid ? 'is-invalid' : ''} validation-without-icon ${darkMode && 'dark-theme-placeholder'
```
Comment (arpitnath) [inline frontend/src/Editor/Components/TextInput.jsx:109]: What is/is the change here?

File: frontend/src/Editor/WidgetManager/widgetConfig.js [hunk 5]
```diff
@@ -879,18 +879,33 @@ export const widgets = [
...
-        validation: { schema: { type: 'string' } },
+        validation: { 
+          schema: { type: 'string' }, 
+        },
       },
       borderRadius: {
         type: 'code',
         displayName: 'Border radius',
-        validation: { schema: { type: 'union', schemas: [{ type: 'string' }, { type: 'number' }] } },
+        validation: {
```
Comment (arpitnath) [inline frontend/src/Editor/WidgetManager/widgetConfig.js:889]: Please revert back to formatting changes.
And try to follow the repository's Lint rules.

**Review by arpitnath — CHANGES_REQUESTED** [review]
Both the files have lint issues, please revert those changes.
And before making changes, please use our lint rules and commit the changes.
You can configure the ESLint for ToolJet from [here](https://docs.tooljet.com/docs/contributing-guide/troubleshooting/eslint)

**Comment by dmdbilal** [comment]
Can you assign this issue to someone. Because my laptop was crashed yesterday so i can't contribute for next 1 month.

**Comment by arpitnath** [comment]
https://github.com/ToolJet/ToolJet/pull/4302#issuecomment-1273236766

Okay, closing the PR

**Pull request closed without merging**